src/app/shared/services/provider/provider.service.ts

```diff
@@ -59,9 +55,8 @@ export class ProviderService {
    * This method update Provider
    * @param Provider
    */
-  updateProvider(provider: Provider): Observable<object> {
-    this.isRelease2 = this.store.selectSnapshot<FeaturesList>(MetaDataState.featuresList).release2;
-    return this.isRelease2 ? this.updateProviderV2(provider) : this.updateProviderV1(provider);
+  updateProvider(provider: Provider, isRelease2: boolean): Observable<object> {
```

Comment:
Can we find out what the data model is here? for Observable

Reply:
Take a look on previous reply.